### DISCUSSION TITLE
Can I install MOOSE on Windows 10?

### DISCUSSION THREAD

**Discussion by sreejitsarmac** [idaholab/moose#25434] [Q&A Getting Started]
Looking to get started. Is it possible to install the software on my Windows 10 PC?

**Comment by GiudGiud** [comment]
Hello

yes you can on windows subsystem for Linux (wsl)

follow the Linux install on the website once you have wsl runninh

**Reply by milljm** [reply]
Reference:
https://mooseframework.inl.gov/getting_started/installation/windows.html

**Reply by GraphicsMonster** [reply]
Hey, I've managed to install and set up vsxsrv but in other guides they want me clone the repo and in this one it's just a few more terminal commands in the ubuntu wsl environment and that got me confused. Which one am I supposed to follow?

**Reply by GiudGiud** [reply]
Follow our instructions.

They do include cloning the repo inside wsl
